src/renderer/state.ts

```diff
@@ -208,6 +208,34 @@ export class AppState {
     versions: this.baseVersions,
   });
 
+  // Lock on the active Electron version that prevents other windows from removing it
```

Comment:
Maybe add a note that this is stored but not used, it only needs to be stored to prevent it from being garbage collected.

Reply:
TBH I wasn't thinking about GC here, I'm just using it to determine whether there's an active lock. Might as well use a boolean flag instead 🤔